Documentation/grpc-metadata.md

```diff
@@ -77,7 +77,8 @@ func (s *server) SomeRPC(ctx context.Context, in *pb.SomeRequest) (*pb.SomeRespo
 
 ## Sending and receiving metadata - client side
 
-Client side metadata sending and receiving examples are available [here](../examples/features/metadata/client/main.go).
+Client side metadata sending and receiving examples are available
```

Comment:
I'm commenting here instead of on line 67 because Github doesn't allow me to comment on unchanged lines.

Is the section titled `Retrieving metadata from context` useful? We are anyways going over this in the section for the server side, and `FromIncomingContext` works only on the server side.

Reply:
Yes, it seems redundant as we repeat that in `Sending and receiving metadata - server side` section